Modules/ConfigOptions.lua

```diff
@@ -123,6 +123,10 @@ return {
 	{ var = "BrandsInLastQuarter", type = "check", label = "Last 25% of Attached Duration?", ifCond = "BrandLastQuarter", apply = function(val, modList, enemyModList)
 		modList:NewMod("Condition:BrandLastQuarter", "FLAG", true, "Config")
 	end },
+	{ label = "Carrion Golem:", ifSkill = "Summon Carrion Golem" },
+	{ var = "carrionGolemNearbvMinion", type = "count", label = "# of Nearby Non-Golem Minions:", ifSkill = "Summon Carrion Golem", apply = function(val, modList, enemyModList)
```

Comment:
Typo: "carrionGolemNearbvMinion" instead of "carrionGolemNearbyMinion".

Reply:
Fixed